stable/dotnetcore/compile-function.sh

```diff
@@ -1,5 +1,14 @@
 #!/bin/bash
 
```

Comment:
What if you use the following [set attributes](https://wiki.bash-hackers.org/commands/builtin/set#attributes):
* `set -e` to exit on the first error.
* `set -x` to show each command before executing it.

This would avoiding multiple calls to the `exit_on_error`.